File: d12Mon/C++/program/p00check_id/CheckId.cpp
#include <iostream>
using namespace std;

class CheckId{
  public:
    string id;
    string pw;
    CheckId(string id, string pw){
      this->id = id;
      this->pw = pw;
    }
    void checkId(){
      if(this->id == "jin" && this->pw == "111"){
        cout << "yeahyeah";
      } else{
        cout << "nopenope";
      }
    }
};

File: d09Fri/TypeScript/program/p00check_id00/CheckId.ts
class CheckId{
  private _id:string = "";
  private _pw:string = "";
  public set id(value: string){
    this._id = value;
  }
  public get id(){
    return this._id;
  }
  public set pw(value: string){
    this._pw = value;
  }
  public get pw(){
    return this._pw;
  }
  public checkId(){
    if(this._id === "jin" && this._pw === "111"){
      console.log("y");
    } else{
      console.log("n");
    }
  }
}
export default CheckId;